crates/hir-ty/src/lib.rs

```diff
@@ -1022,15 +1022,6 @@ pub fn known_const_to_ast(
     db: &dyn HirDatabase,
     display_target: DisplayTarget,
 ) -> Option<ConstArg> {
-    if let ConstValue::Concrete(c) = &konst.interned().value {
```

Comment:
Why is this removed?

Anyway, with this removed, that function can return `ConstArg` instead of `Option<ConstArg>`.

Reply:
> Why is this removed?

`InTypeConstId` is gone now so there isn't anything to match on here